module-new/src/components/dialog/dialog.stories.tsx

```diff
@@ -22,51 +24,124 @@ const Template: StoryObj<typeof Dialog> = {
 
     return (
       <>
-        <Button onClick={() => setDialogOpen(true)}>Open dialog</Button>
-        <Dialog {...args} open={dialogOpen} onOpenChange={setDialogOpen} />
+        <Button testId="dialog-opener" onClick={() => setDialogOpen(true)}>
+          Open dialog
+        </Button>
+        <Dialog testId="dialog-component" {...args} open={dialogOpen} onOpenChange={setDialogOpen} />
       </>
     );
   },
 };
 
+const testTitle = 'Test Dialog';
+const testDesc = 'Hello, I am a test dialog';
```

Comment:
If you're using these consistently, you can set them as args in your Template and they'll spread onto the other components. For the no title test, I think you can just put a replacement args block in.